test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,30 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+
+        // empty list
+        assertNoNull();
+
+        // all objects are not null
+        assertNoNull("abc", 1, 2);
+        assertNoNull("", "");
+
+        assertHasNull((Object) null);
+        assertHasNull(null, null);
+        assertHasNull(null, 1);
+        assertHasNull("abc", null);
+
+    }
+
+    private void assertHasNull(Object... objects) {
```

Comment:
Extracting the assert function is not really necessary here since this method is quite simple (with only one line). Thus, you can add it directly into your test cases.